<launch>
	<arg name="map_location" value="/home/smaug/Desktop/Raw Data/no_ground/one_north_noground_0.pcd" />
	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find pointcloud_registration)/icp_localize.rviz" />
	<node pkg="pointcloud_registration" type="icp_node" name="icp_localize" output="screen">
		<param name="map_location" value="$(arg map_location)" />
		<param name="base_frame_id" value="iMiev/base_link" />
		<param name="odom_frame_id" value="iMiev/odom" />
		<param name="global_frame_id" value="iMiev/map" />
		<param name="icp_iteration" value="5" />
		<param name="icp_distance" value="5.0" />
		<param name="icp_filter_size" value="0.5" />
		<remap from="input" to="/rslidar_points" />
	</node>

	<node pkg="pointcloud_registration" type="show_map" name="show_map" output="screen">
		<param name="map_location" value="$(arg map_location)" />
		<param name="global_frame_id" value="iMiev/map" />
	</node>
	

	
</launch>
